Source/WebCore/accessibility/AXSearchManager.cpp

```diff
@@ -242,6 +242,7 @@ static void appendChildrenToArray(Ref<AXCoreObject> object, bool isForward, RefP
         // We should only ever hit this case with a live object (not an isolated object), as it would require startObject to be ignored,
         // and we should never have created an isolated object from an ignored live object.
         // FIXME: This is not true for ENABLE(INCLUDE_IGNORED_IN_CORE_AX_TREE), fix this before shipping it.
+        // FIXME: We hit this ASSERT on google.com.
```

Comment:
Should we at least understand what's going on here before merging this?

Reply:
Sure, can investigate! Shouldn't be too hard to figure out what's going on here.

Reply:
I added some logging, and confirmed this branch does result in different behavior. All of this code (including the ASSERT) was written at a time when isolated objects could not be ignored, hence this assert. But that assumption is now wrong with `ENABLE(INCLUDE_IGNORED_IN_CORE_AX_TREE)`, so we need to fix this, though the fact that we don't handle this right doesn't cause any readily apparent problems on google.com. I did file a bug to track this so we can address in a follow-up.